code/modules/multiz/structures.dm

```diff
@@ -0,0 +1,281 @@
+//////////////////////////////
+//Contents: Ladders, Stairs.//
+//////////////////////////////
+
+/obj/structure/ladder
```

Comment:
we already have ladder objects, so be careful here because this will overwrite them if you aren't changing the path; this will break existing vaults probably

Reply:
Renamed for disambiguity